qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/RecoveryIT.java

```diff
@@ -258,7 +217,7 @@ public void testRelocationWithConcurrentIndexing() throws Exception {
                 break;
             case UPGRADED:
```

Comment:
I'm aware that during the `MIXED` branch of this switch we index the same documents twice and that might not be great. But it *is* simple. If we don't like it we can change it though.